packages/fetchai/skills/simple_aggregation/strategy.py

```diff
@@ -69,7 +70,7 @@ def __init__(self, **kwargs: Any) -> None:
         self._round = 0
         self._peers = set()  # type: Set[Address]
         self._observations = dict()  # type: Dict[Address, Dict[str, Any]]
-        self._aggregation = None  # type: Optional[Dict[str, Any]]
+        self._aggregation = None  # type: Optional[Any]
```

Comment:
Is it Any? or can it be more tightly defined?

Reply:
Needed to make the typing compatible with the statistics package here, but probably we can do a cast with a more specific definition.

Reply:
ok I see